ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -15,7 +15,7 @@ extension RACScheduler: DateSchedulerType {
 	}
 
 	public func schedule(action: () -> ()) -> Disposable? {
-		return self.schedule(action)
+		return (self.schedule(action) as RACDisposable) as Disposable?
```

Comment:
Hm, why's it different? Isn't it identical? Name transformations:

```
protocol Disposable {}
class RACDisposable: Disposable {}

class RACScheduler {
  func schedule() -> RACDisposable! {
    return RACDisposable()
  }
}

let disposable: Disposable? = RACScheduler().schedule()
```

I guess what I'm trying to show is: this kind of covariant coercion to l-val types is the norm in Swift. e.g. we do things like this all the time:

```
zip(someFunctionReturningAnArray(), aFunctionReturningASequence()
```

The return values from those functions are being transparently covariantly cast to `SequenceType`.